data/json/terrain.json

```diff
@@ -1881,6 +1890,7 @@
         "type" : "terrain",
         "id" : "t_chaingate_l",
         "name": "locked wire gate",
+        "description": "A gate for a chain link fence but this one has a padlock on the latch system.  With the right tools, cutting the metal fence is possible as well as picking the lock.  Or you could just examine the fence to see if it looks climbable.",
```

Comment:
`this one has a padlock on the latch system` is not contrary to `a gate for a chain link fence`. Consider `A gate for a chain link fence. This one has a padlock on the latch system`

Reply:
Might also recommend noting that the gate is locked, which is what's of primary concern here